src/quo/components/text_combinations/page_top/view.cljs

```diff
@@ -0,0 +1,144 @@
+(ns quo.components.text-combinations.page-top.view
+  (:require [clojure.string :as string]
+            [quo.components.avatars.channel-avatar.view :as channel-avatar]
+            [quo.components.avatars.collection-avatar.view :as collection-avatar]
+            [quo.components.inputs.address-input.view :as address-input]
+            [quo.components.inputs.recovery-phrase.view :as recovery-phrase]
+            [quo.components.inputs.search-input.view :as search-input]
+            [quo.components.markdown.text :as text]
+            [quo.components.tags.context-tag.view :as context-tag]
+            [quo.components.text-combinations.page-top.style :as style]
+            [quo.theme]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]
+            [utils.number]))
+
+(defn- format-counter
+  [n]
+  (let [num (utils.number/parse-int n)]
+    (if (<= num 9)
+      (str "0" num)
+      (str num))))
+
+(defn- header-counter
```

Comment:
What is the purpose of this counter?

Reply:
Match the component styles:

![image](https://github.com/status-im/status-mobile/assets/90291778/64a57335-91a1-440e-81dc-a33c2b97ab62)

that `00/00`